pbr/ensure_dependencies.py

```diff
@@ -9,27 +9,27 @@ def _install_pip():
     import requests
     import sys
 
+    install_file = os.path.join(os.path.dirname(__file__), "get_pip.py")
+
     # Get the get-pip.py install file
     print(
         "Pip is not installed, installing it now, restart the script after this is done"
     )
     try:
         r = requests.get("https://bootstrap.pypa.io/get-pip.py", stream=True)
         if r.status_code == 200:
-            with open("get_pip.py", "wb") as f:
+            with open(install_file, "wb") as f:
                 for chunk in r:
                     f.write(chunk)
     except:
-        if not os.path.isfile("get_pip.py"):
+        if not os.path.isfile(install_file):
             print("We were unable to download https://bootstrap.pypa.io/get-pip.py")
             print(
-                "Download it manually and put at {}".format(
-                    os.path.join(os.path.dirname(__file__), "get_pip.py")
-                )
+                "Download it manually and put at {}".format(install_file)
             )
 
     # We need to clobber sys.argv so we don't confuse get-pip.py
-    sys.argv = []
+    sys.argv = [""] if os.name == "nt" else []
```

Comment:
Had an issue with the empty argv on Windows.

Reply:
LGTM